examples/llcstat.yaml

```diff
@@ -0,0 +1,48 @@
+programs:
+  # See:
+  # * https://github.com/iovisor/bcc/blob/master/tools/llcstat.py
+  # * https://github.com/iovisor/bcc/blob/master/tools/llcstat_example.txt
+  - name: llcstat
+    metrics:
+      counters:
+        - name: llc_references_total
+          help: Last level cache operations by type
+          table: references
+          labels:
+            - name: cpu
+              decoders:
+                - name: uint64
+        - name: llc_misses_total
+          help: Last level cache operations by type
+          table: misses
+          labels:
+            - name: cpu
+              decoders:
+                - name: uint64
+    perf_events:
+      - type: 0x0 # HARDWARE
+        name: 0x3 # PERF_COUNT_HW_CACHE_MISSES
```

Comment:
would be nice to use string here and map it back to enum type in go.

Reply:
I looked for a way to reflect a package and resolve constants by name, but couldn't find one.

We can probably make a map ourselves, but I worry that there are platform specific constants.